src/algorithm/butterflies.rs

```diff
@@ -1100,28 +1115,24 @@ impl<T: FftNum> Butterfly13<T> {
     }
 
     #[inline(never)]
-    unsafe fn perform_fft_contiguous(
-        &self,
-        input: RawSlice<Complex<T>>,
-        output: RawSliceMut<Complex<T>>,
-    ) {
+    unsafe fn perform_fft_butterfly<U: LoadStore<Complex<T>> + ?Sized>(&self, buffer: &mut U) {
         // This function was derived in the same manner as the butterflies for length 3, 5 and 7.
         // However, instead of doing it by hand the actual code is autogenerated
         // with the `genbutterflies.py` script in the `tools` directory.
```

Comment:
These bigger prime-sized routines are generated by a script, mentioned here in this comment. One thing missing from this PR is updating that script to generate correct implementations in the future.

Reply:
I took a look at that script and it is currently generating code that was out of date before my PR (i.e. `*buffer.get_unchecked`). I've updated it and believe it is producing the correct output.

Reply:
Oops lol. Unfortunately we don't have any automation making sure they're up to date. Thanks for updating it.